src/components/_global/BalTextInput/composables/useInputStyles.ts

```diff
@@ -0,0 +1,93 @@
+import { Ref, computed } from 'vue';
+
+export default function useInputStyles(props, isInvalid: Ref<boolean>, attrs) {
+  const extPaddingClass = (): string => {
+    switch (props.size) {
+      case 'sm':
+        return 'p-1';
+      case 'lg':
+        return 'p-3';
+      default:
+        return 'p-2';
+    }
+  };
+
+  const intPaddingClass = (): string => {
+    switch (props.size) {
+      case 'sm':
+        return 'p-px';
+      case 'lg':
+        return 'p-2';
+      default:
+        return 'p-1';
+    }
+  };
+
+  const inputTextSize = (): string => {
+    switch (props.size) {
+      case 'sm':
+        return 'text-base';
+      case 'lg':
+        return 'text-2xl';
+      default:
+        return 'text-xl';
+    }
+  };
+
+  const inputHeightClass = (): string => {
+    switch (props.size) {
+      case 'sm':
+        return 'h-8';
+      case 'lg':
+        return 'h-12';
+      default:
+        return 'h-10';
+    }
+  };
```

Comment:
I have set these so that the default size of the inputs is correct for the trade / invest interfaces. sm & lg sized inputs may not be appropriate and so we'll have to adjust these classes as we make more use of the component.